onnx/backend/test/runner/__init__.py

```diff
@@ -168,7 +167,7 @@ def tests(self):  # type: () -> Type[unittest.TestCase]
             onnx_backend_tests = BackendTest(backend).tests
         '''
         tests = self._get_test_case('OnnxBackendTest')
-        for items_map in sorted(self._filtered_test_items.values()):
+        for items_map in sorted(self._filtered_test_items.values()):  # type:ignore
```

Comment:
nit: sync all `# type: ignore` with the same space (there are 4 places)
```suggestion
        for items_map in sorted(self._filtered_test_items.values()):  # type: ignore
```
Thank you for the update! mypy upgrade seems OK for ONNXRUntime as well. I hope we can forward this PR soon.

One question: Do you know why the same function here need to be ignored for the updated mypy?